src/labels.py

```diff
@@ -11,6 +11,8 @@ class Labels:
     TRAFFIC_SIGN = 5
     TRAFFIC_LIGHT = 6
     CAR = 7
+    CITY_BENCH = 8
```

Comment:
Zullen we de hierarchical structure al verwerken in deze branch?

Reply:
We kunnen nog even wachten tot we de road parts en stoplichten ook toevoegen.

Reply:
Waarom CITY_BENCH en niet gewoon BENCH?

Reply:
Maakt het iets duidelijker. Sofa klinkt ook niet als het juiste woord. Zag dit in een paper